eng/Signing.props

```diff
@@ -16,4 +16,12 @@
       -->
     <ItemsToSign Include="$(ArtifactsPackagesDir)**\*.zip" />
   </ItemGroup>
+  <ItemGroup Condition="'$(SignAllBinaries)' == 'true'">
+    <!--
+      Sign all binaries in the artifacts directory.
+      -->
+    <ItemsToSign Remove="@(ItemsToSign)" />
+    <ItemsToSign Include="$(ArtifactsDir)bin\**\*" />
```

Comment:
Will this sign json files properly?

Reply:
No, these are not currently signed either in this new mechanism or in the existing nupkgs. As a follow up, I can look into seeing if signing the json files (and possibly the xml files) (1) actually works (it should), and (2) if the runtime is okay with *.deps.json and *.runtimeconfig.json files being signed.